calendar-cli.py

```diff
@@ -33,6 +33,10 @@
 import logging
 import sys
 import re
+import urllib3
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
I think this logic should only apply if the --ssl-verify-cert option is set to 'no'.

Reply:
You are right, But I don't know how :-)

Reply:
See the bottom of the script, there is some logic on connecting to a caldav server, but only if --nocaldav is not set:

        if not args.nocaldav:
            caldav_conn = caldav_connect(args)
        else:
            caldav_conn = None

Directly below you can add similar logic for disabling warnings:

    if args.ssl_verify_cert == 'no':
        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)

(completely untested code)